### PR TITLE
Moved Http playright test cases to use Https

### PR DESCRIPTION
**Pull request by SrushtiPasari94**

### CODE REVIEW

File: .github/workflows/playwright.yml [hunk 1]
```diff
@@ -32,6 +32,11 @@ jobs:
         node-version: lts/*
     - name: Install dependencies
       run: npm ci
+    - name: Install .NET HTTPS Development Certificate
+      # if: matrix.os == 'ubuntu-latest'
+      run: |
+        dotnet dev-certs https --clean
```
Comment (davidfowl) [inline .github/workflows/playwright.yml:38]: dev certs don't work on linux. Did you test this? https://github.com/tmds/linux-dev-certs

Comment (SrushtiPasari94) [reply .github/workflows/playwright.yml:38]: Was just trying as linux cert was also not working in CI. Undoing this change

Comment (SrushtiPasari94) [reply .github/workflows/playwright.yml]: Used Linux cert

File: playwright.config.ts [hunk 2]
```diff
@@ -22,7 +22,7 @@ export default defineConfig({
   /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
   use: {
     /* Base URL to use in actions like `await page.goto('/')`. */
-    baseURL: 'http://localhost:5045',
+      baseURL: 'http://localhost:5045',
```
Comment (jamesmontemagno) [inline playwright.config.ts:25]: @SrushtiPasari94 should we update this to https endpoing?

Comment (SrushtiPasari94) [reply playwright.config.ts:25]: This also redirect to https.
Somehow with https playwright test are failing. So kept this as is and added middleware for httpsredirect.

Comment (jamesmontemagno) [reply playwright.config.ts:25]: gotcha!

**Comment by jamesmontemagno** [comment]
Updated and tested out mobile clients and seem to be working well here.

**Review by DamianEdwards — APPROVED** [review]
This looks like a good incremental improvement. We should look at making all the projects use HTTPS in a follow-up PR.

**Pull request merged**